public-dashboard-client/src/page-sentencing/PageSentencing.js

```diff
@@ -92,9 +114,18 @@ export default function PageSentencing() {
           some point after their release.
         </>
       ),
+      otherControls: (
```

Comment:
I really like how these pages are structured, and the `DetailPage` is so flexible to to handle each of these different pages content. It seems like it will scale very well to many pages and many states with potential copy/visualization differences between them. Well done.

Reply:
funny, this feature has been making me realize how annoying I find it to work with `DetailPage` but I guess that's why it's good to get outside perspectives! 

It's a bit messy and rigid in practice IMO but that was the tradeoff for encapsulating what otherwise would have been a ton of boilerplate on all of these pages. (It has been making me think about how nice it would be to set up all of that UI state in some data models instead of having all these components bootstrap themselves with it ... 🚀 )